rpc/common/service.proto

```diff
@@ -27,7 +27,7 @@ message PackageInfo {
 message Application {
   string           type      = 1;
   string           file_path = 2;
-  repeated Package libraries = 3;
+  repeated Package packages  = 3;
```

Comment:
IIRC it's better to use new number, even if we just rename the field.

Reply:
Renaming should be ok as protobuf internally uses number, not name. I'll check it just in case.